src/scripts/components/about-project/about-project.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {FunctionComponent} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import styles from './about-project.styl';
+
+const AboutProject: FunctionComponent = () => (
+  <div className={styles.aboutProject}>
+    <div className={styles.title}>
+      <h1>
+        <FormattedMessage id="about" />
+      </h1>
+    </div>
+    <div className={styles.description}>
+      <FormattedMessage id="projectDescription" />
```

Comment:
The about page should be markdown right?

Reply:
I thought we agreed on having it in the translations. 🤔

Reply:
Yes, but with markdown support :)

Reply:
Ah, I see! Thanks, will add it.